src/compiler/test-file/formats/es-next.js

```diff
@@ -2,10 +2,11 @@ import loadBabelLibs from '../../load-babel-libs';
 import APIBasedTestFileCompilerBase from '../api-based';
 
 const BABEL_RUNTIME_RE = /^babel-runtime(\\|\/|$)/;
+const FLOW_MARKER_RE   = /^\s*\/\/\s*@flow|^\s*\/\*\s*@flow\s*\*\//;
```

Comment:
First part of this condition will match anything that starts with `//@flow`, e.g. `//@flowwhatever`.